pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -1422,6 +1422,16 @@ func ValidateVirtualMachineSpec(field *k8sfield.Path, spec *v1.VirtualMachineSpe
 			Field:   field.String(),
 		})
 	}
+
+	// Validate RunStrategy
+	if spec.Running && spec.RunStrategy != nil {
```

Comment:
Well, you can switch midstream actually--if you edit the VM resource. The real touchy part is in virtctl--where the patch logic needs to account for what's already been used.

The thing I had indecision about here is whether or not "Running" should become a pointer to a boolean, rather than direct reference.